packages/components/src/FormField/FormField.tsx

```diff
@@ -146,145 +137,158 @@ export interface FormFieldProps {
+        (hasMiniLabel || type === "time" || type === "select") && placeholder,
+      [styles.invalid]: invalid || error,
+    },
+  );
+
+  const Wrapper = inline ? "span" : "div";
+
+  const labelClassNames = classnames(
+    styles.label,
+    type === "textarea" && styles.textareaLabel,
+  );
+
+  return (
+    <>
+      <Wrapper
+        className={wrapperClassNames}
+        style={{ ["--formField-maxLength" as string]: maxLength || max }}
+      >
+        <label className={labelClassNames} htmlFor={identifier}>
+          {placeholder || " "}
+        </label>
+        {fieldElement()}
+        {type === "select" && (
+          <span className={styles.icon}>
+            <Icon name="arrowDown" />
+          </span>
         )}
-
-        {validations && !inline && <InputValidation messages={validations} />}
-
-        <Wrapper
-          className={wrapperClassNames}
-          style={{ ["--formField-maxLength" as string]: maxLength || max }}
-        >
-          <label className={labelClassNames} htmlFor={identifier}>
-            {placeholder || " "}
-          </label>
-          {fieldElement()}
-          {type === "select" && (
-            <span className={styles.icon}>
-              <Icon name="arrowDown" />
-            </span>
-          )}
-        </Wrapper>
-      </>
-    );
-
-    function fieldElement() {
-      const fieldProps = {
-        id: identifier,
-        className: styles.formField,
-        name: name,
-        disabled: disabled,
-        readOnly: readonly,
-        onChange: handleChange,
-        value: value,
-        ...(defaultValue && { defaultValue: defaultValue }),
-      };
-
-      switch (type) {
-        case "select":
-          return <select {...fieldProps}>{children}</select>;
-        case "textarea":
-          return (
-            <textarea
-              rows={rows}
-              onFocus={handleFocus}
-              onKeyDown={event => handleKeyDown(event)}
-              onBlur={handleBlur}
-              ref={ref as Ref<HTMLTextAreaElement>}
-              {...fieldProps}
-            />
-          );
-        default:
-          return (
+      </Wrapper>
+      {error && !inline && <InputValidation message={error} />}
+    </>
+  );
+
+  function fieldElement() {
+    const fieldProps = {
+      id: identifier,
+      className: styles.formField,
+      name: name,
+      disabled: disabled,
+      readOnly: readonly,
+      onChange: handleChange,
+      value: value,
+      ...(defaultValue && { defaultValue: defaultValue }),
+    };
+
+    switch (type) {
+      case "select":
+        return <select {...fieldProps}>{children}</select>;
+      case "textarea":
+        return (
+          <textarea
+            rows={rows}
+            onFocus={handleFocus}
+            onKeyDown={event => handleKeyDown(event)}
+            onBlur={handleBlur}
+            ref={e => {
+              /**
+               * @typescript-eslint/ban-ts-ignore is ignored as `.current` is readonly
```

Comment:
Would this be required if inputRef was a `MutableRefObject`? Or does that not work for other reasons?

Reply:
I tried `MutableRefObject` and it causes errors in other files. Example is in `InputText` we get errors for things like `.blur() is not assignable`.